src/util.ts

```diff
@@ -53,25 +139,68 @@ export function findFirst<T, K extends keyof T, V extends T[K]>(
             return object;
         }
     }
-
     return null;
 }
 
+/**
+ * Generates the next id.
+ *
+ * @param objs the map of already exist elements with ids
+ * @return the next available id
+ */
+export function getNextId<T extends Map<string, V>, V>(objs: T): string {
+    let i = objs.size;
+    while (objs.has(i.toString())) {
+        i++;
+    }
+    return i.toString();
+}
+
+/**
+ * Adds review details to a project paper.
+ *
+ * @param paper the project paper to extend with review information
+ * @return the new project paper with an added 'reviews' array populated from the associated reviews
+ */
+export function addProjectPaperReviews(paper: ServerProjectPaper): Project_Paper {
+    return {
+        ...paper,
+        reviews: PAPER_REVIEWS.get(paper.id)!.map((reviewId) => REVIEWS.get(reviewId)!),
+    };
+}
+
+/**
+ * Checks if any property in an object (including nested objects) is undefined.
+ *
+ * @param obj the object to check
+ * @returns true if any property (or sub-property) is undefined, otherwise false
+ */
 export function anythingUndefined<T extends object>(obj: T): boolean {
     return Object.values(obj).some((v) => {
         return v == undefined || (typeof v === "object" && anythingUndefined(v));
     });
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function isSnowballRService(methodDescriptor: ServerMethodDefinition<any, any>): boolean {
-    return methodDescriptor.path.startsWith("/snowballr.SnowballR/");
-}
-
-export function getNextId<T extends Map<string, V>, V>(objs: T): string {
-    let i = objs.size;
-    while (objs.has(i.toString())) {
-        i++;
+/**
+ * Checks whether the given string indicates that an option is enabled.
+ * An option is considered as enabled, if it is set to either:
+ * - 1
+ * - (y/Y)es
+ * - (t/T)rue
+ *
+ * @param option the input to check
+ * @return true, if the given option is set to value listed above, otherwise false
+ */
+export function isOptionEnabled(option?: string): boolean {
+    option = option?.toLowerCase() ?? "";
+    switch (option) {
+        case "1":
+        case "yes":
+        case "Yes":
+        case "true":
+        case "True":
+            return true;
+        default:
+            return false;
```

Comment:
The cases `Yes` and `True` are not reachable as you do `toLowerCase`.
```suggestion
    option = option?.toLowerCase() ?? "";
    switch (option) {
        case "1":
        case "yes":
        case "true":
            return true;
        default:
            return false;
```
This could also be `return ["1", "yes", "true"].include(option);`